package.json

```diff
@@ -97,18 +97,18 @@
     "express-session": "1.17.1",
     "filesize": "6.1.0",
     "form-data": "3.0.0",
-    "govuk-frontend": "3.8.1",
+    "govuk-frontend": "3.9.0",
```

Comment:
I would do this as a separate update so that it can be marked as a `feat` change

Reply:
Any particular reason? If we _always_ want to do that I can also add this package to the list of ignore updates for now?

Reply:
I'll restate what I said ;)

I think upgrading `govuk-frontend` is worth a line in the release notes (just as `moj-frontend` would be). The other dependencies are not of interest to the business, but as `govuk-frontend` can change the look and feel and/or behaviour...

ie:  `feat(app): Upgrade govuk-frontend to 3.9.0`

Reply:
Agreed to disagree offline - proceeding as is